src/System/Random/Internal.hs

```diff
@@ -105,6 +103,8 @@ import GHC.ForeignPtr
 import Data.ByteString (ByteString)
 #endif
 
+#include "MachDeps.h"
```

Comment:
We can avoid CPP: there is `GHC.ByteOrder` plus `ghc-byteorder` package for old GHCs:
```
  if impl(ghc < 8.4)
    build-depends:  ghc-byteorder
```

Reply:
Thank you for the suggestion. Done.